src/Components/login/materialStyles.js

```diff
@@ -0,0 +1,23 @@
+const messageSignInStyle = {
```

Comment:
we should in clude the `textFieldBorderStyle` within this file, I dont see it here